test/e2e/e2e_suite_test.go

```diff
@@ -28,8 +28,8 @@ import (
 	_ "github.com/triggermesh/triggermesh/test/e2e/framework"
 
 	// test suites
-	_ "github.com/triggermesh/triggermesh/test/e2e/sources"
-	_ "github.com/triggermesh/triggermesh/test/e2e/targets"
+	// _ "github.com/triggermesh/triggermesh/test/e2e/sources"
```

Comment:
There is a simpler way to work on a single test: use `FDescribe()` inside your Ginkgo test.
"F" stands for Focus, it runs only the focused series of tests.